cli/pact.js

```diff
@@ -0,0 +1,118 @@
+/*jslint node: true */
+'use strict';
+
+/**
+ * Spawns the skyux pact command.
+ * @name pact
+ */
+function pact(command, argv) {
+  const logger = require('../utils/logger');
```

Comment:
I would suggest organizing this variable declaration section better.  Typically I have my `const`s that require external modules, then internal, then `let`s